src/routes/fileRoutes.ts

```diff
@@ -1,12 +1,323 @@
 import express from 'express';
-import { updateFile, getFileById, getAllFiles, uploadFile } from '../controllers/fileController';
+import { updateFile, getFileById, getAllFiles, uploadFile, deleteFile } from '../controllers/fileController';
 
 const router = express.Router();
 
+/**
+ * @swagger
+ * components:
+ *   securitySchemes:
+ *     bearerAuth:
+ *       type: https
+ *       scheme: bearer
+ *       bearerFormat: JWT
+ *
+ *   schemas:
+ *     File:
+ *       type: object
+ *       properties:
+ *         id:
+ *           type: integer
+ *           example: 10
+ *         title:
+ *           type: string
+ *           example: "Prova Algoritmos III - 2023/1"
+ *         path:
+ *           type: string
+ *           example: "https://cdn.pixabay.com/photo/2023/03/18/07/55/tawny-owl-7860086_960_720.jpg"
+ *         postId:
+ *           type: integer
+ *           example: 4
+ *         commentId:
+ *           type: integer
+ *           example: 25
+ *       required:
+ *         - title
+ *         - path
+ *         - postId
+ *         - commentId
+ *
+ */
+
+/**
+ * @swagger
+ * /api/file/uploadFile:
+ *   post:
+ *     security:
+ *       - bearerAuth: []
+ *     description: Upload new file.
+ *     tags: [File]
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             $ref: '#/components/schemas/File'
+ *
+ *           example:
+ *             title: Prova Algoritmos III - 2023/1
+ *             path: https://cdn.pixabay.com/photo/2023/03/18/07/55/tawny-owl-7860086_960_720.jpg
+ *             postId: 3
+ *             commentId: 1
+ *     responses:
+ *       200:
```

Comment:
I think that it should be "201" because something was created.
```suggestion
 *       201:
```